### PR TITLE
Optimize spend tutorial for data retention based pricing

### PR DESCRIPTION
**Pull request by jakerachleff**

### CODE REVIEW

File: versioned_docs/version-2.0/tutorials/Administrators/manage_spend.mdx [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+The usage graph lets us examine how much of each usage based pricing metric we have consumed lately. It does not give insight
+into spend. 
+
+We can navigate to the Usage Graph under `Settings` -> `Usage and Billing` -> `Usage Graph`. 
+
+![](./static/p1usagegraph.png)
+
+We see in the graph above that there are two usage metrics that LangSmith charges for, LangSmith Traces (Base Charge) and 
+LangSmith Traces (Extended Data Retention Upgrades). The first metric tracks all traces that you send to LangSmith. The second
+tracks all traces that have our Extended 400 Day Data Retention. For more details, see [TODO: LINK]
```
Comment (jakerachleff) [inline versioned_docs/version-2.0/tutorials/Administrators/manage_spend.mdx:34]: Where do we think we should put the docs on how auto upgrades work? It feels almost like the pricing docs need to expand to usage/pricing, and multiple sections now. data retention based pricing, auto upgrades, usage limits, etc all need explanation in some places

File: versioned_docs/version-2.0/tutorials/Administrators/manage_spend.mdx [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+
+### Change project level retention defaults
+Our existing projects have not changed their data retention settings, so we can change these on the individual project pages. 
+
+We navigate to `Projects` -> `<your project name>`, click the data retention drop down, and modify it to base retention. As 
+with the organization level setting, this will only affect retention (and pricing) for traces going forward.
+
+![](./static/p1projectretention.png)
+
+### Optional: keep around 10% of traces for extended data retention
```
Comment (jakerachleff) [inline versioned_docs/version-2.0/tutorials/Administrators/manage_spend.mdx:78]: Note: this doesn't work yet, I have to code this into existence tonight

Comment (jakerachleff) [reply versioned_docs/version-2.0/tutorials/Administrators/manage_spend.mdx:78]: right now the rule doesn't run

Comment (jakerachleff) [reply versioned_docs/version-2.0/tutorials/Administrators/manage_spend.mdx:78]: all the data is fake :)

File: versioned_docs/version-2.0/tutorials/Administrators/manage_spend.mdx [hunk 3]
```diff
@@ -0,0 +1,157 @@
...
+:::note
+The extended data retention limit can cause features other than traces to stop working once reached. If you plan to 
+use this feature, please read more about its functionality here.
+:::
+
+## Summary
+In this tutorial, we learned how to:
+
+1. Cut down our existing costs with data retention policies
+2. Prevent future overspend with usage limits
```
Comment (jakerachleff) [inline versioned_docs/version-2.0/tutorials/Administrators/manage_spend.mdx:155]: Should we add a section (3) around cleaning up run rules that have high sampling rates?

**Comment by barberscott** [comment]
Consolidating the pricing and rate limit changes here as well since they all go together.

**Review by langchain-infra — APPROVED** [review]

**Pull request merged**